src/esn.inbox.libs/app/services/mailboxes/mailboxes-service.spec.js

```diff
@@ -1563,7 +1564,7 @@ describe('The inboxMailboxesService factory', function() {
         });
       };
       jmapClient.mailbox_changes = function() {
-        return $q.when({ oldState: '1', newState: '2' });
+        return $q.when({ oldState: '1', newState: '2', hasMoreChanges: true });
```

Comment:
`hasMoreChanges` should not be true if `mailbox_get` return the same state than changes.newState `'2'`.